### DISCUSSION TITLE
Updated Quire from v0.18.0 and the images repeat again in zoom mode

### DISCUSSION THREAD

**Discussion by Badragan** [thegetty/quire#207] [Get Help]
Had the same problem with v0.18.0. If there were more than 2-3 images on a page, in zoom mode some of them were shown more than once, replacing others that were not shown at all. I was then sent two new q-figure.html and q-figure-group.html to replace the existing ones. This solved the issue. (it had something to do with UnixNano)

I tried to use again the same 2 files, but the resulting behaviour is not very nice. The "zoom" square icon moves under the image and it looks bad (see attached figure). I tried to replicate the changes that were sent initially, but the new q-figure files have a different content and I couldn't find where to try the changes.

Did anybody else encounter the issue and find a solution?

(sorry, I have no experience with this html/js/css coding, I am just trying to use the platfor
![imgIssue](https://user-images.githubusercontent.com/56792169/141214049-fa64ccb8-4ef7-42a7-ae9c-832d1685356b.jpg)

**Comment by geealbers** [comment]
Hi @Badragan, thanks for posting about this. We're seeing the issue as well and our developers will be working on a fix. I'll post again here as soon I have something.

**Reply by Badragan** [reply]
Thank you!
